proposals/all_proposals.md

```diff
@@ -0,0 +1,671 @@
+Public places and institutions are likewise able to foster and implement more democratic technologies. For instance libraries can set up their systems in a more open source secure way while at the same time spreading knowledge and best practices to its users. A good example for this is the American Library Freedom Project (https://libraryfreedomproject.org/)
+
+Public media, public advertisement
+
+Another place to push back the dominance of corporate power and abuse of digital technologies by other players is to oppose their all encompassing presence in media and advertising (TV, Internet, Public sphere). There have to be state- or EU funded campaigns advocating for the above mentioned solutions and their essential importance for a free, democratic and also economically more just society - think platform and surveillance capitalism where, corporations mine data to generate exorbitant surplusses, without paying taxes and drastically changing “the game” for unaware citizens respectively, in corporate talk, users.
+
+<lynX> Let's forbid unethical surveillance capitalism rather than de-facto accepting it by running (desperate) campaigns for the "alternatives."
+
+# 4 
+# 30.06.2018 Input DSC Vienna1 - 18-06-30
+
+With the rapidly rising importance of science and technology, the importance of their democratic control is also growing. Most of basic research is still financed by the public, either directly done by public institutions are in private institutions but partly funded via public grants. Only when it comes to making final products out of this, this is done mostly in the private realm. There, most patent applications are done. Thus, most of the costs for research are paid by the public while most gains are privatized. The current patent-system is extremely harmful. The most logical conclusion would be to abolish the patent-system all together: This way everyone can profit from the common knowledge.
+<lynX> Patents also have their useful aspects. I would suggest to work on societal taxes and dividends for redistribution, this way the gains the corps had through public research flows back to the citizen. See also 'European New Deal'.
+
+As we can see, there is a trend towards big monopoly power within the tech industry: Google, Microsoft, Apple, Facebook, Amazon. The reason for this is, that with these new technologies cooperation is a much more efficient way of production then competition. So these large corporations have the advantage of being able to have cooperation within their company. 
+<lynX> This is not an accurate description on how the big 5 became monopolies.
+
+Of course the monopoly power is extremely harmful and dangerous. Thus we need to find ways to promote cooperative production without the need for mega-corporation. One extremely successful model here is open-source/free software: It encourages cooperation: while everyone can benefit from the existing base of software, the model also encourages the contribution to this pool.
```

Comment:
Free/Open Source software was one of the winning ingredients of Google and subsequent monopolists. When it came to making a product they moved the code to the private realm, just like it is described with research above. FLOSS alone is not the solution, it needs supporting legislation. - lynx